integration-cli/docker_cli_events_test.go

```diff
@@ -18,13 +18,12 @@ import (
 )
 
 func (s *DockerSuite) TestEventsTimestampFormats(c *check.C) {
-	image := "busybox"
+	name := "events-time-format-test"
 
 	// Start stopwatch, generate an event
-	time.Sleep(1 * time.Second) // so that we don't grab events from previous test occurred in the same second
 	start := daemonTime(c)
-	dockerCmd(c, "tag", image, "timestamptest:1")
-	dockerCmd(c, "rmi", "timestamptest:1")
+	time.Sleep(1 * time.Second) // so that since > start
```

Comment:
not quite follow, how since > start here, as found below
`since, until := f(start), f(end)`
seems since = $start